api/views/search.py

```diff
@@ -1,30 +1,33 @@
 from api import app, db
-from api.models import Story, StoryPOI, POI
-from api.utils import create_response, row_constructor
+from api.models import POI
+from api.utils import create_response
 from flask import Blueprint, request
 
-SEARCH_URL = "/search"
+SEARCH_POIS_URL = "/search/pois"
 
 mod = Blueprint('search', __name__)
 
-@app.route(SEARCH_URL, methods=['GET'])
+@app.route(SEARCH_POIS_URL, methods=['GET'])
 def search_pois():
     data = request.args
-    pois = []
-    pois1 = []
-    pois2 = []
+    pois_by_name = []
+    pois_by_description = []
 
     if data.get('q') is not None:
         if data.get('name') is not None:
-            if data.get('name') == "True":
-                pois1 = POI.query.filter(POI.name.contains(data['q']))
+            if data.get('name') == "true":
+                pois_by_name = POI.query.filter(POI.name.contains(data['q']))
+        else:
+            return create_response(status=404, message='Missing name parameter (true/false)')
         if data.get('description') is not None:
-            if data.get('description') == "True":
-                pois2 = POI.query.filter(POI.description.contains(data['q']))
+            if data.get('description') == "true":
+                pois_by_description = POI.query.filter(POI.description.contains(data['q']))
+        else:
+            return create_response(status=404, message='Missing description parameter (true/false)')
+    else:
+        return create_response(status=404, message='Missing query parameter')
 
-    if pois1.count() == 0 and pois2.count() == 0:
-        return create_response(status=404, message='No POIs found')
-    list1 = [p.to_dict() for p in pois1]
-    list2 = [p.to_dict() for p in pois2]
-    pois = list1 + list2
-    return create_response({'pois_list': pois})
+    pois_by_name_list = [p.to_dict() for p in pois_by_name]
+    pois_by_description_list = [p.to_dict() for p in pois_by_description]
+    pois = pois_by_name_list + pois_by_description_list
```

Comment:
this is just me being really picky, but naming the variables `..._list` is kinda redundant since it's very clear that they are lists, since they are defined by a list comprehension

Reply:
i'm not sure what you're suggesting here.. should we name it something else?
I saw that in the last set of PR comments you said to try combining them into one statement but it doesn't work because pois_by_name and pois_by_description are queries and can't be combined directly, which is the purpose of having these 3 lines to begin with. 
i'm just going to leave this as is for now